localstack_services.sh

```diff
@@ -83,10 +83,18 @@ printf "${greenColor}...Setting up S3${reset}\n"
 cd $basedir/env/cloud/s3
 terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
 
+printf "${greenColor}...Setting up S3 Datalake Buckets${reset}\n"
+cd $basedir/env/cloud/buckets
+terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
+
```

Comment:
I was just trying to keep things independent for now, so that I could cut out things as necessary and not get lost in pre-existing scripts. I'd look at moving them into the s3 module prior maybe - but I also like the idea of not tightly coupling so if we need to nuke or change it can be quick.

Lemme know thoughts!~ :)

Reply:
I also like the idea of keeping things independent. What we could do is to just put it in the S3 module but have separate `.tf` files to identify the new buckets as being part of the Datalake work.